src/overlays/actors/ovl_En_In/z_en_in.c

```diff
@@ -21,92 +45,1572 @@ const ActorInit En_In_InitVars = {
+                        this->actionFunc = func_808F3B40;
+                        func_800B8A1C(&this->actor, globalCtx, 0x92, 500.0f, 100.0f);
+                        ret = 1;
+                    } else {
+                        func_800E8EA0(globalCtx, &this->actor, 0x34A2);
+                        ret = 0;
+                    }
+                    break;
+                case 0x34A2:
+                    func_800E8EA0(globalCtx, &this->actor, 0x34A3);
+                    ret = 0;
+                    break;
+                case 0x34A3:
+                    func_808F43E0(this);
+                    func_80151BB4(globalCtx, 0x11);
+                    ret = 1;
+                    break;
+                case 0x3499:
+                    func_800E8EA0(globalCtx, &this->actor, 0x349A);
+                    func_808F30B0(&this->skelAnime, 1);
+                    func_808F30B0(&this->unk4A4->skelAnime, 7);
+                    ret = 0;
+                    break;
+                case 0x349A:
+                    func_800E8EA0(globalCtx, &this->actor, 0x349B);
+                    ret = 0;
+                    break;
+                case 0x349B:
+                    if (msgCtx->choiceIndex == 0) {
+                        func_808F4270(globalCtx, this, arg2, msgCtx, 1);
+                        ret = 0;
+                    } else {
+                        func_8019F230();
+                        func_800E8EA0(globalCtx, &this->actor, 0x349C);
+                        ret = 0;
+                    }
+                    break;
+                case 0x349C:
+                    func_808F43E0(this);
+                    func_808F35D8(this, globalCtx);
+                    func_80151BB4(globalCtx, 0x11);
+                    ret = 1;
+                    break;
+            }
+            break;
+        case 6:
+            switch (textId) {
+                case 0x3496:
+                    func_808F4108(this, globalCtx, 0x3497);
+                    ret = 0;
+                    break;
+                case 0x3498:
+                case 0x34A6:
+                    func_800E8EA0(globalCtx, &this->actor, 0x3471);
+                    ret = 0;
+                    break;
+                case 0x3471:
+                    func_808F4270(globalCtx, this, arg2, msgCtx, 0);
+                    ret = 0;
+                    break;
+                case 0x3472:
+                    func_808F43E0(this);
+                    gSaveContext.weekEventReg[56] &= (u8)~8;
+                    func_80151BB4(globalCtx, 0x11);
+                    ret = 1;
+                    break;
+                case 0x349E:
+                    func_808F4108(this, globalCtx, 0x349F);
+                    gSaveContext.weekEventReg[56] &= (u8)~8;
+                    ret = 0;
+                    break;
+            }
+            break;
+    }
+    return ret;
+}
+
+s32 func_808F5674(GlobalContext* globalCtx, EnIn* this, s32 arg2) {
+    s32 pad;
+    s32 ret = 0;
```

Comment:
```suggestion
    s32 ret = false;
```
same for this function